haystack/document_store/base.py

```diff
@@ -214,3 +223,59 @@ def delete_documents(self, index: Optional[str] = None, filters: Optional[Dict[s
     def run(self, documents: List[dict], index: Optional[str] = None, **kwargs):  # type: ignore
         self.write_documents(documents=documents, index=index)
         return kwargs, "output_1"
+
+    @abstractmethod
+    def get_documents_by_id(self, ids: List[str], index: Optional[str] = None,
```

Comment:
I think should `get_documents_by_ids`

Reply:
@lalitpagaria Every document store using this method name. If we need to change all those method names too.

Reply:
Yeah for now please ignore this comment for this PR